.ci/Dockerfile

```diff
@@ -10,9 +10,9 @@ RUN sudo apt-get update && \
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add - && \
     sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-12 main" && \
     sudo apt-get update && \
-    sudo apt-get install --yes --allow-unauthenticated libffi7 && \
-    sudo apt-get install --yes --allow-unauthenticated libllvm12 && \
-    sudo apt-get install --yes --allow-unauthenticated libclang-cpp12 && \
-    sudo apt-get install --yes --allow-unauthenticated clang-format-12 && \
+    sudo apt-get install --yes --allow-unauthenticated libffi7 libllvm12 libclang-cpp12 clang clang-format-12
+
+RUN sudo apt-get install --yes cmake ninja-build pkg-config libgtk-3-dev && \
```

Comment:
Why do we split this into a separate RUN?

Reply:
It doesn't need to be, but I separated it because the ones in the previous run all have to do with LLVM, and getting them from the LLVM repo. I figured it makes sense to group them because if they don't change, we don't need to rebuild that chunk of the image, and the rebuild of the Docker image can go faster, since it'll be cached.